src/wing/TiglWingHelperFunctions.cpp

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2019 CFS Engineering
+ *
+ * Created: 2019 Malo Drougard <malo.drougard@protonmail.com>
+ * Author: Malo Drougard
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+#include "TiglWingHelperFunctions.h"
+
+#include "tigl.h"
+#include "CCPACSWingSegment.h"
+
+namespace tigl
+{
+
+namespace winghelper
+{
+
+bool HasShape(const tigl::CCPACSWing& wing)
+{
+    return wing.GetSegmentCount() > 0;
+
+}
+
+TiglAxis GetDeepDirection(const tigl::CCPACSWing& wing)
```

Comment:
I think this function deserves its own unit test.

Reply:
I agree